code/extensions/che-commands/tests/empty.spec.ts

```diff
@@ -0,0 +1,15 @@
+/**********************************************************************
+ * Copyright (c) 2022 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+/* eslint-disable header/header */
+
+describe('no-op', () => {
+  it('no-op', () => { });
```

Comment:
I don't think it's a good idea to replace existing tests by `no-op` stub

Reply:
Nothing to test here as I have removed `utils.ts` at all and have just added the stub to be aligned with other modules.
But I do not like it as well. So removed it at all.